content/en/docs/reference/node/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Node Reference Information
+weight: 40
+---
```

Comment:
Ah, you might want to wait for https://github.com/kubernetes/website/pull/31758 to merge, and then rebase your branch @adrianreber. Although Git ought to work things out, it's cleaner to rely purely on the existing commits to _main_ here.

Reply:
I will rebase once that PR is merged. I just copied from the main for now to have a working PR.

Reply:
#31758 has merged in now.

Reply:
Thanks. git handled it automatically after a rebase. The change is gone from the PR. Marking as resolved.